ml_metadata/tools/mlmd_bench/stats.cc

```diff
@@ -20,6 +20,8 @@ limitations under the License.
 #include "absl/time/clock.h"
 #include "absl/time/time.h"
 #include "ml_metadata/metadata_store/types.h"
+#include "ml_metadata/tools/mlmd_bench/proto/mlmd_bench.pb.h"
+#include "tensorflow/core/platform/logging.h"
```

Comment:
drop this?

Reply:
Since Report() now takes a `WorkloadConfigResult& workload_summary`, we may need to include `mlmd_bench.pb.h`. Also, I included `logging.h` for `LOG(ERROR) << "Current workload has not been executed even once!";` when `done_ == 0` cases in `Report()`. wdyt?

Reply:
I removed the `#include "tensorflow/core/platform/logging.h"` but the compiler complains. I still removed it and along with the `LOG(ERROR)` below since num_ops == 1 may not be an error if a user specifies so.